swagger.yaml

```diff
@@ -645,20 +645,27 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
-  "/severa/users/{severaUserId}/flextime":
+  "/users/{userId}/flextime":
     get:
       operationId: getFlextimeBySeveraUserId
       summary: Get the flextime balance for a user.
       description: Retrieve the flextime balances for a user based on their SeveraUserId.
       tags:
         - FlexTime
       parameters:
-        - name: severaUserId
+        - name: userId
           in: path
           required: true
           description: severaId of the user.
           schema:
             type: string
+        - name: eventDate     
+          in: query
+          required: false
+          description: Date for which to get the balance
+          schema:
+            type: string
+            format: date
```

Comment:
Do we actually need this?

Reply:
You removed the eventDate in the lambda as it was unused, so I'm not sure why you would add it back in here?